### DISCUSSION TITLE
Label sur les idées

### DISCUSSION THREAD

**Discussion by detobel36** [Kodi-vStream/venom-xbmc-addons#3450] [Idées]
Salut,

Je propose de mettre des labels sur les idées... Du genre:
- Appliqué
- Refusé
- En réflexion
- Manque de détails

Et du coup, je pense qu'on peut supprimer le label "idée" (ou au moins dire dans la description qu'il est "obsolète" et qu'il faut aller dans les discussions. Dans cette même idée, on peut rajouter un label "déplacé vers discussions".

**Comment by sizanic** [comment]
Salut,

Je pense que tu parles de la suppression du label "idée" **sur les issues**, pas ici pour les discussions.
Oui, c'est une bonne idée.

**Comment by detobel36** [comment]
Yep c'est ça...

Label != Catégorie 😉
